envi/archs/i386/disasm.py

```diff
@@ -158,6 +158,9 @@ def getOperValue(self, op, emu=None):
         if emu is None: return None # This operand type requires an emulator
         return emu.getRegister(self.reg)
 
+    def getOperAddr(self, op, emu=None):
+        return None
```

Comment:
and here, i'd expect an address to a be an `int`, so if this returns a non-int, then my script might break. recommend that this raise an error, too, to indicate that the routine was invoked in a way that doesn't make sense.

Reply:
if you call getOperAddr without expecting a None, your scripts are already breaking.  perhaps that's not the ideal situation, but it is the expected behavior.  look at all the other getOperAddr's in each architecture.  this is the default behavior for non-derefs.